src/chapters/sql-part2/exercises.rst

```diff
@@ -0,0 +1,97 @@
+Exercises: SQL, Part 2
+=======================
+
+In the previous chapter, we set up a database to store information about seeds. Let's expand that idea now that we know all about foreign keys!
+
+Create a New Schema
+-------------------
+
+To get started, let's set up a new schema in MySQL Workbench and call it ``garden``.
+In the last chapter, we learned we can do this by clicking on *New Schema*, naming it ``garden``, accepting the defaults, and clicking *Apply*.
+
+Import New Tables
+-----------------
+
+To import the data in our CSVs, we need to create three empty tables.
+
+Our first table is ``plant``. ``plant`` needs to have four columns:
+
+#. ``plant_id``, which will be the primary key for our database.
+#. ``plant_name``, which will contain the name of the plant.
+#. ``zone`` is an integer for the hardiness zone that our plant will live in best.
```

Comment:
"where our plant" instead of "that our plant".

Reply:
Done! Thanks!